apps/schema/static/schema/swagger.yaml

```diff
@@ -176,6 +171,61 @@ paths:
               allOf:
               - $ref: '#/components/requestBodies/shipment/createAttributes'
 
+  /api/v1/shipments/{shipment_id}/tracking/:
+    post:
+      summary: Add tracking data
+      description: >
+        Associate GPS tracking data with a `Shipment`.
+      parameters:
+      - $ref: '#/components/parameters/shipment/path'
+      tags:
+      - Shipments
+      responses:
+        '204':
+          description: *response_204
+        '401':
+          description: *response_401
+          content:
+            application/vnd.api+json:
+              schema:
+                $ref: '#/components/schemas/errors/401'
+      requestBody:
+        content:
+          application/json:
+            schema:
+              allOf:
+              - $ref: '#/components/requestBodies/shipment/tracking/payload'
+          multipart/form-data:
+            schema:
+              allOf:
+              - $ref: '#/components/requestBodies/shipment/tracking/payload'
+    get:
+      summary: Get tracking data
+      description: >
+        Associate GPS tracking data with a `Shipment`.
```

Comment:
Another copy/paste error.

Please give an appropriate description and also make a note that the content returned in the 'data' attribute of the response conforms to the GeoJSON spec (link to https://tools.ietf.org/html/rfc7946).